azure/templates/pds-tests.yml

```diff
@@ -131,12 +119,12 @@ steps:
       export TEST_PATIENT_ID="$(TEST_PATIENT_ID)"
       export INTERNAL_DEV_ASID="$(INTERNAL_DEV_ASID)"
 
-      poetry run pytest -v tests/functional/test_patient_create.py --api-name=personal-demographics-service --proxy-name "$(FULLY_QUALIFIED_SERVICE_NAME)" --junitxml=tests/patient-create-test-report.xml --reruns 20 --reruns-delay 1
-    displayName: Run post patient tests
+      poetry run pytest -v tests/functional/test_patient_access.py --api-name=personal-demographics-service --proxy-name "$(FULLY_QUALIFIED_SERVICE_NAME)" --junitxml=tests/patient-create-test-report.xml --reruns 20 --reruns-delay 1
```

Comment:
Shouldn't the --junitxml part be left as patient-access-test-report.xml?

Reply:
It would make sense, but it really makes no difference, the report gets published all the same and this test will be removed eventually anyway.